src/librustdoc/visit.rs

```diff
@@ -1,5 +1,11 @@
 use crate::clean::*;
 
+/// allows a type to traverse the cleaned as of a crate.
```

Comment:
Nit:

```suggestion
/// Allows a type to traverse the cleaned AST of a crate.
```

Reply:
you want me to capitalize this sentence specifically? and not any of the other doc comments?

i was just matching the style of the existing comments.

Reply:
Sorry ignore the capitialization if it's pre-existing, just the as -> AST

Reply:
will do!